scripts/sct_image.py

```diff
@@ -584,7 +584,7 @@ def orientation(im, ori=None, set=False, get=False, set_data=False, verbose=1, f
             try:
                 printv('Get orientation...', verbose)
                 im_out = None
-                ori = get_orientation(im)
+                ori = im.orientation
```

Comment:
Here the orientation is accessed vs. computed before.

Before, we would be evaluating the orientation from the header.

Cases I see:

1. Image was copied, in which case the original header wasn't touched -> same behaviour as before, as now orientation which is used, has also been copied over

2. Image was loaded, in which case orientation has been set by loadFromPath.

Is there another case? If not, then no problem.